core/serialisation/src/test/java/uk/gov/gchq/gaffer/serialisation/util/JsonSerialisationUtilTest.java

```diff
@@ -184,6 +184,34 @@ public void testClassWithTypeParamExtendingComparable() {
         assertEquals(expectedValues.entrySet(), result.entrySet());
     }
 
+    @Test
+    public void testClassWithArray() {
+        // Given
+        final String className = ClassWithArray.class.getName();
+
+        // When
+        final Map<String, String> result = JsonSerialisationUtil.getSerialisedFieldClasses(className);
+
+        // Then
+        final Map<String, String> expectedValues = new HashMap<>();
+        expectedValues.put("array", String[].class.getCanonicalName());
+        assertEquals(expectedValues.entrySet(), result.entrySet());
```

Comment:
You could use assertJ here for a cleaner test. Example:
```java
assertThat(result).containsExactly(entry("array", String[].class.getCanonicalName()));
```
Anywhere I have seen the pattern of creating an expectedValue data structure and doing assertEquals, I have been trying to use assertJ instead.

Reply:
This could also be:
```java
assertThat(result).containsEntry("array", String[].class.getCanonicalName());
```
but to check that it only contains that, you would also have to add a `.hasSize(1)`.

Reply:
See my other comment about assertj.